.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
+omit =
+    authors/*__init__.py
+    */tests/*
+    */migrations/*
+     authors/settings.py
+     */wsgi.py
+
+source = 
+      authors
```

Comment:
You will also need to omit the unnecessary files when the `report` command is  run. something like
```bash
[report]
omit= <files & folders to be omitted>
```
This will ensure the coverage percentage only reflects cover of actual functionality.